HdProduction.HelpDesk.Api/Models/Tickets/TicketResponseModel.cs

```diff
@@ -1,7 +1,12 @@
+using HdProduction.HelpDesk.Domain.Entities;
+
 namespace HdProduction.HelpDesk.Api.Models.Tickets
 {
-  public class TicketResponseModel : TicketRequestModel
-  {
-    
-  }
+    public class TicketResponseModel : TicketRequestModel
+    {
+        public long Id { get; set; }
+        public TicketAttribute TicketStatus {get;set;}
```

Comment:
1st - why do you use type TicketAttribute for properties, if it is possible to use concrtete type (TicketStatus, TicketPriority etc.)?
2nd - ok, if you use it like TicketAttribute, than use TicketAttributeResponse model at least